.gitignore

```diff
@@ -0,0 +1 @@
+\.idea
```

Comment:
nit: Since this is a folder, could change it to be `.idea/`?

Reply:
Yes, it can be chaged. I have already done it.